src/app/components/app-layout/app-layout.component.html

```diff
@@ -37,15 +37,18 @@
     <div *ngIf="isAuthenticated" class="profile">
       <div class="btn-group" dropdown>
         <button class="btn dropdown-toggle" dropdownToggle id="button-basic" type="button"
-                aria-controls="dropdown-basic">
-          {{ name }} <img alt="" class="profile-picture" src="assets/janette-doe.png"> <span class="caret"></span>
+          aria-controls="dropdown-basic">
+          {{ name }} <img alt="" class="profile-picture" src="assets/janette-doe.png"> <span
```

Comment:
Imo the images should be in a subfolder in assets instead of being in the root of assets. I purpose we put images in the following folder `assets/images`

Reply:
Definitely, changed all the images in the entire project.